packages/plugin-js-packages/src/lib/config.unit.test.ts

```diff
@@ -8,18 +8,6 @@ import {
 } from './config.js';
 
 describe('jsPackagesPluginConfigSchema', () => {
-  it('should accept a JS package configuration with all entities', () => {
-    expect(() =>
-      jsPackagesPluginConfigSchema.parse({
-        auditLevelMapping: { moderate: 'error' },
-        checks: ['audit'],
-        packageManager: 'yarn-classic',
-        dependencyGroups: ['prod'],
-        packageJsonPaths: ['./ui-app/package.json', './ui-e2e/package.json'],
-      } satisfies JSPackagesPluginConfig),
-    ).not.toThrow();
-  });
```

Comment:
I don't see why we shouldn't keep this test. Why not just adjust `packageJsonPath`?

Reply:
We can, I deleted it as it felt too similar to the one bellow, but this one uses just `prod` and different package manager, I'll add it back